voxblox_ros/include/voxblox_ros/intensity_vis.h

```diff
@@ -0,0 +1,42 @@
+#ifndef VOXBLOX_ROS_THERMAL_VIS_H_
```

Comment:
Fix header guards. Did you disable the linter? There are quite a few things in this PR that shouldn't have gotten through the linter :| Like the signed/unsigned comparisons, lots of missing system includes, ref to temporary, header guards, etc.